src/Compare/ExtensionMethodReflection.php

```diff
@@ -0,0 +1,47 @@
+<?php
```

Comment:
This is virtually identical code to ExtensionConfigReflection.php, should use a superclass or trait